client/tx/tx.go

```diff
@@ -193,9 +201,6 @@ func Sign(txf Factory, name string, txBuilder client.TxBuilder, overwriteSig boo
 		// use the SignModeHandler's default mode if unspecified
 		signMode = txf.txConfig.SignModeHandler().DefaultMode()
 	}
-	if err := checkMultipleSigners(signMode, txBuilder.GetTx()); err != nil {
```

Comment:
`checkMultipleSigners` was added to avoid using the CLI for signing multi-signers Msgs with DIRECT, it's a bad UX: you need a 1st step where all signers choose their SIGN_MODE and put inside AuthInfo, gather all that, and then a 2nd step where all signers sign over that AuthInfo.

So you're probably right, we probably should keep `checkMultipleSigners`. However it must be modified, because with tips, we want to allow the fee payer to use DIRECT in a multi-signer tx (the other signer being the tipper).

Maybe we can add a check that with multiple signers, only the last signer can be DIRECT. That way we still avoid the multi-DIRECT bad UX, but allow tipping.